eslint.js

```diff
@@ -31,19 +32,19 @@ class EslintTask extends TaskKitTask {
 
     const results = cli.executeOnFiles(this.options.files).results;
     let hasError = false;
+
     results.forEach((result) => {
       if (result.errorCount > 0) {
         this.log(['error'], formatter(results));
         hasError = true;
-      }
-      if (result.warningCount > 0) {
+      } else if (result.warningCount > 0) {
```

Comment:
Don't you want to log both errors and warnings?

Reply:
This was logging both. Just twice. One prepended with error, and one with warning, but exact same content. Since the formatter doesn't seem to be able to output separately (only errors separatedly but not warnings)

<img width="763" alt="screenshot 2018-10-29 18 01 18" src="https://user-images.githubusercontent.com/946645/47666774-b6a22800-dba4-11e8-9953-5e03860265ab.png">